main.tf

```diff
@@ -0,0 +1,234 @@
+        "securityhub:GetFindings",
+        "securityhub:ListEnabledProductsForImport"
+      ]
+      resources = ["*"]
+    }
+  }
+
+  dynamic "statement" {
+    for_each = local.multiple_account_id
+
+    content {
+      effect = "Allow"
+      actions = [
+        "organizations:DescribeOrganization",
+        "organizations:ListAccounts",
+        "organizations:DescribeAccount",
+        "iam:ListAccountAliases",
+        "ec2:DescribeRegions",
+        "securityhub:ListFindingAggregators",
+        "securityhub:GetFindings",
+        "securityhub:ListEnabledProductsForImport"
+      ]
+      resources = ["*"]
+    }
+  }
+}
+
+module "task_role" {
+  source  = "schubergphilis/mcaf-role/aws"
+  version = "~> 0.4.0"
+
+  name                  = local.ecs_task_iam_name
+  create_policy         = true
+  permissions_boundary  = null
+  principal_identifiers = ["ecs-tasks.amazonaws.com"]
+  principal_type        = "Service"
+  role_policy           = data.aws_iam_policy_document.ecs_task.json
+  tags                  = merge({ "Name" = local.ecs_task_iam_name }, var.tags)
+}
+
+module "task_execution_role" {
+  source  = "schubergphilis/mcaf-role/aws"
+  version = "~> 0.4.0"
+
+  name                  = local.ecs_task_execution_iam_name
+  create_policy         = true
+  permissions_boundary  = null
+  principal_identifiers = ["ecs-tasks.amazonaws.com"]
+  principal_type        = "Service"
+  role_policy           = data.aws_iam_policy.ecs_task_execution.policy
+  tags                  = merge({ "Name" = local.ecs_task_execution_iam_name }, var.tags)
+}
+
+module "task_events_role" {
+  source  = "schubergphilis/mcaf-role/aws"
+  version = "~> 0.4.0"
+
+  name                  = local.ecs_events_iam_name
+  create_policy         = true
+  permissions_boundary  = null
+  principal_identifiers = ["ecs-tasks.amazonaws.com"]
+  principal_type        = "Service"
+  role_policy           = data.aws_iam_policy.ecs_events.policy
+  tags                  = merge({ "Name" = local.ecs_events_iam_name }, var.tags)
+}
+
+resource "aws_cloudwatch_event_rule" "default" {
+  name                = local.name
+  schedule_expression = var.schedule_expression
+}
+
+resource "aws_cloudwatch_event_target" "default" {
+  target_id = local.name
+  arn       = local.cluster_arn
+  rule      = aws_cloudwatch_event_rule.default.name
+  role_arn  = module.task_events_role.arn
+
+  ecs_target {
+    launch_type         = "FARGATE"
+    task_count          = 1
+    task_definition_arn = aws_ecs_task_definition.default.arn
+    platform_version    = "1.4.0"
+
+    network_configuration {
+      assign_public_ip = false
+      security_groups  = [aws_security_group.default[0].id]
+      subnets          = var.subnet_ids
+    }
+  }
+}
+
+module "aws_ecs_container_definition" {
+
+  source  = "terraform-aws-modules/ecs/aws//modules/container-definition"
+  version = "~> 5.11.4"
+
+  cloudwatch_log_group_name       = "/aws/ecs/${local.name}"
```

Comment:
can you make a var out of `/aws/ecs/` and default them to `/aws/ecs/` some customers need a custom prefix

Reply:
consistent with all the other mcaf module atm. So leaving this for a future Pr to fix if someone needs it.